src/settings.ts

```diff
@@ -64,10 +66,26 @@ export class SettingTab extends PluginSettingTab {
                 .setValue(this.plugin.settings.recreateTree)
                 .onChange(async (value) => {
                     this.plugin.settings.recreateTree = value;
+                    this.display();
                     await this.plugin.saveSettings();
                 }
             ));
         
+        if (this.plugin.settings.recreateTree) {
+            new Setting(containerEl)
+                .setName('Remove Path')
+                .setDesc('Remove parts of the path, separate in the settings by comma or new line.')
```

Comment:
This feature is a little difficult to explain, but hopefully this may be slightly clearer?
```suggestion
                .setName('Remove Folders From Path')
                .setDesc('Removes the specified folders from the output path, if present. Separate folders by using a comma or a new line. Names are case insensitive.')
```